libs/designsystem/src/lib/components/tabs/tab-button/tab-button.component.scss

```diff
@@ -41,16 +46,34 @@ ion-tab-button {
 
   @include utils.media('>=medium') {
     flex: none;
+    flex-direction: row;
     padding: 0 utils.size('m');
     margin-right: utils.size('m');
+    font-size: utils.font-size('s');
+
+    --color-selected: #{utils.get-color('black')};
+
+    // Magic values here as we have no tokens that gives
+    // badge the correct position in relation to the tab icon
+    --kirby-badge-position: absolute; // Repeated for readability
+    --kirby-badge-top: calc(50% - 1.35em);
+    --kirby-badge-left: 1.6em;
+
+    @include utils.slotted(kirby-badge) {
+      &.md {
+        --kirby-badge-position: relative;
+        --kirby-badge-top: 0;
+        --kirby-badge-left: 0;
+
+        margin-left: utils.size('xxxxs');
+        margin-bottom: math.div(utils.size('xxxxs'), 2);
```

Comment:
Remember to import the math module (beginning of file):

```scss
@use 'sass:math';
```

Will fix https://github.com/kirbydesign/designsystem/actions/runs/3319146598/jobs/5483935259#step:4:36

Reply:
Bummer... - fixed